tests/system/feature_store/test_spark_engine.py

```diff
@@ -692,8 +692,26 @@ def test_different_paths_for_ingest_on_spark_engines(
             if is_parquet
             else CSVTarget(name="tar", path=target_path)
         )
+
+        source_path = (
+            "v3io:///bigdata/test_different_paths_for_ingest_on_spark_engines.parquet"
+        )
+        fsys = fsspec.filesystem(v3iofs.fs.V3ioFS.protocol)
+        stocks.to_parquet(path=source_path, filesystem=fsys)
+        source_parquet = ParquetSource(
```

Comment:
`source_parquet`... Because parquet is French? :)

Reply:
changed :-)